### DISCUSSION TITLE
Metric/Interval is more than 1

### DISCUSSION THREAD

**Discussion by tatsean** [microsoft/perfview#1435] [Q&A]
Hi Guys, I tried to run PerfView in my local machine (Windows 10) to collect some ETW, and here is the result I got (screenshot). However, what I don't understand is why the Metric/Interval is 29.44 (which is much bigger than 1). I understand how the figure is calculated, which basically "Totals Metric" \ "Last-First", but It does not make sense to me why the "Total Metrics" is so high? 

Please advice. 

![perfview_result](https://user-images.githubusercontent.com/487243/119494317-c2bde880-bd93-11eb-8a04-4a2e9e4cd8db.png)

**Comment by brianrob** [comment]
@tatsean, this is expected.  The reason is that you're using a Thread Time view, so this is showing you data across all threads whether or not they are blocked or on the CPU.  If you just limit to CPU time, you still may not see a metric/interval of 1 or less because you have an 8 proc machine, so the range for CPU would be between 0 and 8.

**Reply by tatsean** [reply]
Now, i got it. Thanks @brianrob
